contents/docs/libraries/python/_snippets/contexts.mdx

```diff
@@ -100,7 +100,8 @@ You can read more about identifying users in the [user identification documentat
 
 Contexts can be associated with a session ID by calling `posthog.set_context_session`. Session IDs must be UUIDv7 strings.
 
-> **Using PostHog on your frontend too?** If you're using the PostHog JavaScript Web SDK on your frontend, it generates a session ID for you and you can retrieve it by calling `posthog.get_session_id()` there. We then recommend passing that session ID to your backend by setting the `X-POSTHOG-SESSION-ID` header and extracting it in your request handler (if you're using our Django middleware, this happens automatically).
+> **Using PostHog on your frontend too?** If you're using the PostHog JavaScript Web SDK on your frontend, it generates a session ID for you and you can retrieve it by calling `posthog.get_session_id()`. We then recommend passing that session ID to your backend by setting the `X-POSTHOG-SESSION-ID` header. Alternatively you can enable '__add_tracing_headers' in your config and the header will be set automatically.
```

Comment:
This might look better as a callout box. I find that larger text chunks are better for callout boxes, one liners often do better as comment block + a emoji

https://posthog.com/handbook/content/docs-style-guide#callout-boxes